0005-kzen-two-party-ecdsa.adoc

```diff
@@ -0,0 +1,173 @@
+= Kzen Two Party ECDSA Review
+Lloyd Fournier <lloyd@coblox.tech>
+:toc:
+:revdate: 2019-05-10
+
+* Authors(s): {authors}
+* Date: {revdate}
+
+Issue: https://github.com/comit-network/comit-rs/issues/897[#897]
+
+
+== Context
+
+In order to do a _Scriptless Scripts_ atomic swap <<Sanchez18a>> with ECDSA signatures we need to have an implementation of <<Lindell17>>.
+The point of this spike is to review <<Lindell17>> and KZen's rust implementation <<kzen_mp_ecdsa>> and make notes about.
```

Comment:
I'm missing some kind of conclusion, e.g. is the provided library sufficient for us to be used right away?

Reply:
To me it sounded like it is :) 

But I agree, a conclusion clearly stating that would be good!

Reply:
I added an `== Outcome Summary` at the beginning.